tools/convert_datasets/bdddet2coco.py

```diff
@@ -0,0 +1,79 @@
+import argparse
```

Comment:
This file has been removed by the PR 34.
To get the update, you can do this:
`git merge master; git push origin tao`

Reply:
Before push, also check the running

Reply:
After merging, also add some descriptions at docs/GET_START.md

Reply:
bdd2coco is not relevant to this merge. I just delete those two files.

Reply:
`git pull origin master` is a better option